source/deployment/ldap-group-sync.rst

```diff
@@ -107,6 +109,8 @@ It may take a few seconds to load all team and channel memberships for a user de
 
 .. image:: ../images/Team_Channel_Membership_Sync.png
 
+To manage a team or private channel with synchronized groups, please see `this documentation <https://docs.mattermost.com/deployment/ldap-group-constrained-team-channel.html>`_.
```

Comment:
Thoughts on also adding this to the FAQ of ldap-group-sync.rst, for easier discoverability? 0/5, e.g.

```
How do I manage a team or private channel with synchronized groups?

You can do so with .... See this documentation to learn more
```

Reply:
Great idea.  Added.